pkgs/by-name/an/angle/package.nix

```
@@ -0,0 +1,152 @@
+{
+  stdenv,
+  gn,
+  ninja,
+  llvmPackages_21,
+  gclient2nix,
+  pkg-config,
+  glib,
+  python3,
+  symlinkJoin,
+  lib,
+  xorg,
+  wayland,
+  pciutils,
+  libGL,
+  ...
+}:
+let
+  llvmMajorVersion = lib.versions.major llvmPackages_21.llvm.version;
+  arch = stdenv.hostPlatform.parsed.cpu.name;
+  triplet = lib.getAttr arch {
+    "x86_64" = "x86_64-unknown-linux-gnu";
+    "aarch64" = "aarch64-unknown-linux-gnu";
+  };
+
+  clang = symlinkJoin {
+    name = "clang";
+    paths = [
+      llvmPackages_21.llvm
+      llvmPackages_21.clang
```

Might be best to have a common var for llvm rather than needing to change it in 4 different places in sync

```suggestion
  # specify llvm version
  llvmPackages = llvmPackages_21;
  llvmMajorVersion = lib.versions.major llvmPackages.llvm.version;
  arch = stdenv.hostPlatform.parsed.cpu.name;
  triplet = lib.getAttr arch {
    "x86_64" = "x86_64-unknown-linux-gnu";
    "aarch64" = "aarch64-unknown-linux-gnu";
  };

  clang = symlinkJoin {
    name = "clang";
    paths = [
      llvmPackages.llvm
      llvmPackages.clang
```

Also in nativeBuildInputs